content/site/about-us.md

```diff
@@ -23,31 +23,26 @@ We have various social media accounts where we share updates and new content wit
             <svg width="100" aria-hidden="true" viewBox="0 0 71.41206 50"><defs/><style>.st1{fill:var(--backgroundColor);} .st2{fill:var(--highImpactBlack);}</style><g transform="scale(.58824)"><path class="st2" fill-opacity="1" d="M118.9 13.3c-1.4-5.2-5.5-9.3-10.7-10.7C98.7 0 60.7 0 60.7 0s-38 0-47.5 2.5C8.1 3.9 3.9 8.1 2.5 13.3 0 22.8 0 42.5 0 42.5s0 19.8 2.5 29.2C3.9 76.9 8 81 13.2 82.4 22.8 85 60.7 85 60.7 85s38 0 47.5-2.5c5.2-1.4 9.3-5.5 10.7-10.7 2.5-9.5 2.5-29.2 2.5-29.2s.1-19.8-2.5-29.3z"/><path class="st1" d="M80.2 42.5L48.6 24.3v36.4z"/></g></svg>
         </a>
     </li>
-    
```

Comment:
All I needed to do to fix the SVG being rendered as markdown code was to remove the spacing

Reply:
LOL I hate that this file is this sensitive. My fault, but ugh